src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -242,6 +243,23 @@ export const documentIntegrationList: DocumentIntegration[] = [
       },
     ],
   },
+  {
+    slug: 'octohook',
+    name: 'Octohook',
+    author: 'Octohook',
+    docUrl: 'https://docs.octohook.com/integrations/sentry.html',
+    description:
+      'Octohook is a platform that lets you visualize, debug and redistribute your webhooks.',
+    features: [
+      {
+        featureGate: 'webhook',
```

Comment:
is `webhook` a new feature gate?

Reply:
Yea, none of the existing options made sense to me, so added a new one.

Reply:
can't you technically use this with alert rules too? since that's just another kind of webhook we send

Reply:
Oh yea, fair point. Let me add that.